scripts/install-rkt.sh

```diff
@@ -29,6 +29,16 @@ tar xvzf rkt-v"${version}".tar.gz
 cat <<EOF >install-pak
 #!/bin/bash
 
+# fix mkdir issues with checkinstall and fstrans
+for dir in /usr/lib/rkt/stage1-images/\\
+        /usr/share/man/man1/\\
+        /usr/share/bash-completion/completions/\\
+        /usr/lib/tmpfiles.d/\\
+        /usr/lib/systemd/system/
+do
+    mkdir -p \$dir 2>/dev/null || :
```

Comment:
`|| :` ensures those commands - which will fail due to described bug - exit with the code `0`. One can argue if that is (still) required. I did run the script with `set -e` in order to see where the install-pak fails. The script did bail out on any failed command (`exit_code != 0`) - which is a pretty sane approach IMHO.

mkdir creates the dir in the prefix that checkinstall uses transparently.  But seems to check the un-prefixed directory for existence. `install` does invoke `mkdir`. 

Check for details in #3123 and the bug reports that are linked there.